src/nrc/api/types.py

```diff
@@ -17,3 +17,15 @@ class NotificationMessage(TypedDict):
 
 class SendNotificationTaskKwargs(NotificationMessage):
     notificatie_id: NotRequired[int | None]
+
+
+class CloudEventKwargs(TypedDict):
+    id: str
+    source: str
```

Comment:
I don't know if there is a type for this in `typing`, but according to the spec this should be a `URI-reference`. Ideally the OAS also indicates this, but not sure if such a type exists there either